graphql/schema.graphql

```diff
@@ -69,6 +69,10 @@ type AnnotatedDoc {
 	These words need to be corrected or reviewed further.
 	"""
 	unresolvedForms: [AnnotatedForm!]!
+	"""
+	Collection chapter path of this document
+	"""
+	chapterPath: [String!]
```

Comment:
A document isn't guaranteed to live in just one collection

Reply:
Ah okay, my reasoning for the change was that people who have old documents bookmarked can be redirected to the document's corresponding chapter in CWKW. Should they be kept to the same document page instead?

Reply:
And then just return all of the chapters that this document is attached to. Then on the front-end, we only redirect if there's exactly one chapter the document is within.

Reply:
If there's multiple, we can show a list of links to them. It also wouldn't be that bad to leave the current document pages up for now too. I originally thought we'd have more collection-specific content associated with each document but we don't have _any_ planned so far.